### DISCUSSION TITLE
jynew V0.2 stable version

### DISCUSSION THREAD

**Discussion by chenggongchina** [jynew/jynew#991] [Announcements]
稳定版本V0.2，可以正常体验游戏了

<hr /><em>This discussion was created from the release <a href='https://github.com/jynew/jynew/releases/tag/v0.2-stable'>jynew V0.2 stable version</a>.</em>

**Comment by Hopefulee** [comment]
I do not live in China, I do not have Baidu account. Can you upload to other cloud platform please?

**Reply by chenggongchina** [reply]
Thanks for suggestion, which platform do you suggest?

**Reply by luminchung** [reply]
Perhaps Google Drive would be easily accessible to most? Just my suggestion unless someone else has better idea.

**Reply by MephistoSatanDevil** [reply]
> 

https://sourceforge.net/projects/jynew/files/
try this link

**Comment by luminchung** [comment]
That works,

Thank you so much!

**Reply by ytxcash** [reply]
大佬中午好啊，方便在阿里云上传一份吗？
百度实在是太慢

**Reply by MephistoSatanDevil** [reply]
> 

包体下载里提供的第一个链接不是百度啊
